docs/src/manual/ansatz/mps.md

```diff
@@ -18,36 +19,76 @@ using NetworkLayout
 ```@example viz
 fig = Figure() # hide
 
-tn_open = rand(MatrixProduct{State,Open}, n=10, χ=4) # hide
-tn_periodic = rand(MatrixProduct{State,Periodic}, n=10, χ=4) # hide
+open_mps = rand(MPS; n=10, maxdim=4) # hide
 
-plot!(fig[1,1], tn_open, layout=Spring(iterations=1000, C=0.5, seed=100)) # hide
-plot!(fig[1,2], tn_periodic, layout=Spring(iterations=1000, C=0.5, seed=100)) # hide
+plot!(fig[1,1], open_mps, layout=Spring(iterations=1000, C=0.5, seed=100)) # hide
 
 Label(fig[1,1, Bottom()], "Open") # hide
-Label(fig[1,2, Bottom()], "Periodic") # hide
 
 fig # hide
 ```
 
+### Canonical Forms
+
+An `MPS` representation is not unique: a single `MPS` can be represented in different canonical [`Form`](@ref). The choice of canonical form can affect the efficiency and stability of algorithms used to manipulate the `MPS`. You can check the canonical form of an `MPS` by calling the `form` function:
+
+```@example
+mps = MPS([rand(2, 2), rand(2, 2, 2), rand(2, 2)])
+
+form(mps)
+```
+
+Currently, `Tenet` supports the [`NonCanonical`](@ref), [`CanonicalForm`](@ref) and [`MixedCanonical`](@ref) forms.
+
+#### `NonCanonical` Form
+In the `NonCanonical` form, the tensors in the `MPS` do not satisfy any particular orthogonality conditions. This is the default `form` when an `MPS` is initialized without specifying a canonical form. It is useful for general purposes but may not be optimal for certain computations that benefit from orthogonality.
+
+#### `Canonical` Form
+Also known as Vidal's form, the `Canonical` form represents the `MPS` using a sequence of isometric tensors (`Γ`) and diagonal vectors (`λ`) containing the Schmidt coefficients. The `MPS` is expressed as:
+
+```math
+| \psi \rangle = \sum_{i_1, \dots, i_N} \Gamma_1^{i_1} \lambda_2 \Gamma_2^{i_2} \dots \lambda_{N-1} \Gamma_{N-1}^{i_{N-1}} \lambda_N \Gamma_N^{i_N} | i_1, \dots, i_N \rangle \, .
+```
+
+You can convert an `MPS` to the `Canonical` form by calling `canonize!`:
+
+```@example
+mps = MPS([rand(2, 2), rand(2, 2, 2), rand(2, 2)])
+canonize!(mps)
+
+form(mps)
+```
+
+#### `MixedCanonical` Form
+In the `MixedCanonical` form, tensors to the left of the orthogonality center are left-canonical, tensors to the right are right-canonical, and the tensors at the orthogonality center (which can be `Site` or `Vector{<:Site}`) contains the entanglement information between the left and right parts of the chain. The position of the orthogonality center is stored in the `orthog_center` field.
+
+You can convert an `MPS` to the `MixedCanonical` form and specify the orthogonality center using `mixed_canonize!`:
+
+```@example
+mps = MPS([rand(2, 2), rand(2, 2, 2), rand(2, 2)])
+mixed_canonize!(mps, Site(2))
+
+form(mps)
+```
+
```

Comment:
One can check that the MPS is effectively in mixed canonical form using the functions `isleftcanonical` and `isrightcanonical`, ie 

```
psi = rand(MPS; n=10, maxdim=7)
mixed_canonize!(psi, Site(4))
isleftcanonical(psi, Site(3)) # True 
isrightcanonical(psi, Site(5)) # True 
```

Reply:
we do have a helper function that checks that the MPS respects its form,yes ? eg 
ismixedcanonical(psi) = isleftcanonical(psi) && isrightcanonical(psi) 
where the two take as input the orthogonality center from psi to make the check and all..

Reply:
you should not use `isleftcanonical`/`isrightcanonical` as they are deprecated, but use `isisometry` instead